arches_lingo/src/arches_lingo/pages/SchemeList.vue

```diff
@@ -1 +1,60 @@
-<template>Schemes list placeholder</template>
+<script setup lang="ts">
+import { onMounted, ref } from "vue";
+import SchemeCard from "@/arches_lingo/components/scheme/SchemeCard.vue";
+import { fetchSchemes } from "@/arches_lingo/api.ts";
+
+import type { SchemeDescriptor } from "@/arches_lingo/types";
+
+const schemes = ref<SchemeDescriptor[]>([]);
+
+onMounted(async () => {
+    schemes.value = await fetchSchemes();
+});
+</script>
+
+<template>
+    <Suspense>
+        <div>
+            <ul
+                style="
+                    display: flex;
+                    flex-wrap: wrap;
+                    list-style-type: none;
+                    padding: 0;
+                "
+            >
+                <!-- Create New Scheme -->
+                <li class="scheme-card new-scheme-card">
+                    Placeholder for creating new scheme
+                </li>
+                <!-- Existing Schemes -->
+                <li
+                    v-for="scheme in schemes"
+                    :key="scheme.resourceinstanceid"
+                    class="scheme-card"
+                >
+                    <SchemeCard :scheme="scheme" />
+                </li>
+            </ul>
+        </div>
+    </Suspense>
+</template>
+
+<style scoped>
+.scheme-card {
+    margin: 0.5rem;
+    padding: 1rem;
```

Comment:
clicking the padding in the card does not result in navigation:

<img width="387" alt="Screenshot 2024-12-03 at 2 37 45 PM" src="https://github.com/user-attachments/assets/8e8ee5ac-add7-4433-a1ff-c0fe7103c417">


also arguably the css relating to the card (height, width, border, etc ) should live in the card and not the list.